pkg/server/handler/appHandlers.go

```diff
@@ -26,6 +32,14 @@ func GetApps(c *gin.Context) {
 }
 
 func ListApps(c *gin.Context) {
+	kubeClient := c.MustGet("KubeClient")
+	namespace := c.Param("envName")
```

Comment:
it's env name,not namespace. Namespace is one information contained in env which can retrieve from envfile by envName.

Reply:
Fixed, thanks.